content/soarian-financials/stu3/individuals/patient.md

```diff
@@ -0,0 +1,638 @@
+	"start": "2018-12-14"
+	},
+	"assigner": {
+	"reference": "Organization/118",
+	"display": "Commonwealth of PA"
+	}
+	}
+	],
+	"active": true,
+	"name": [
+	{
+	"use": "OFFICIAL",
+	"text": "James,Smith,Edward",
+	"family": "James",
+	"given": [
+	"Smith",
+	"Edward"
+	],
+	"period": {
+	"start": "2018-12-14"
+	}
+	}
+	],
+	"telecom": [
+	{
+	"extension": [
+	{
+	"valueBoolean": true,
+	"url": "http://hl7.org/fhir/StructureDefinition/iso21090-preferred"
+	}
+	],
+	"system": "PHONE",
+	"value": "(645) 748 4747",
+	"use": "HOME"
+	},
+	{
+	"extension": [
+	{
+	"valueBoolean": false,
+	"url": "http://hl7.org/fhir/StructureDefinition/iso21090-preferred"
+	}
+	],
+	"system": "PHONE",
+	"value": "(547) 457 4848",
+	"use": "MOBILE"
+	},
+	{
+	"system": "EMAIL",
+	"value": "james@cerner.com"
+	}
+	],
+	"gender": "MALE",
+	"birthDate": "2008-12-11",
+	"deceasedBoolean": false,
+	"address": [
+	{
+	"use": "HOME",
+	"type": "POSTAL",
+	"text": "51 Valley Stream parkway,Malvern,PA 19355",
+	"line": [
+	"51 Valley Stream parkway"
+	],
+	"city": "Malvern",
+	"district": "Chester",
+	"state": "PA",
+	"postalCode": "19355",
+	"country": "US",
+	"period": {
+	"start": "2018-12-14"
+	}
+	}
+	],
+	"maritalStatus": {
+	"coding": [
+	{
+	"system": "http://hl7.org/fhir/v3/MaritalStatus",
+	"code": "S",
+	"display": "Single"
+	}
+	],
+	"text": "Single"
+	},
+	"communication": [
+	{
+	"language": {
+	"coding": [
+	{
+	"system": "urn:ietf:bcp:47",
+	"code": "en",
+	"display": "ENGLISH"
+	}
+	],
+	"text": "ENGLISH"
+	},
+	"preferred": true
+	}
+	]
+	}
+
+### Errors
+
+The following common errors and OperationOutcomes may be returned.
+
+Error														 | Severity   | Code 			  | Diagnostics
+-------------------------------------------------------------|------------|-------------------|--------------------------------------------------------------------------------------------------------
+There is no active  legal name 								 | Error      | not-supported 	  | Retrieval of unidentified patient is not supported at this time.
+If the patient being searched for is merged 				 | Error      | not-supported 	  | The patient being searched for has been merged. The merged target patient number is (Target Person  No)
+If the patient being searched for is not found in the system | Error      | not-found 	      | The patient is not found
+
+
+## Create
+
+Create an individual Patient:
+
+    POST /Patient
+
+_Implementation Notes_
+
+* The following elements are not supported and will cause an error/warning to be returned if set:
+    * [Patient.multipleBirth]
+    * [Patient.photo]
+    * [Patient.contact]
+    * [Patient.animal]
+    * [Patient.managingOrganization]
+    * [Patient.link]
+	
+
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', Accept: 'application/json', 'Content-Type': 'application/json'} %>
+
+### Body Fields
+
+#### Resource Type
```

Comment:
use the `<%= definition_table(:patient, :create, :dstu2) %>` macro with the appropriate lib/resources/stu3/soarian_financials_patient.yaml file added